### PR TITLE
Add recipe for python-hookman

### PR DESCRIPTION
**Pull request by nicoddemus**

### CODE REVIEW

File: recipes/python-hookman/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  home: https://github.com/esss/hookman
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: HookMan is a python package that provides a plugin management system to applications, specially those who are written (in totally or partially) in C++.  
+
+extra:
+  recipe-maintainers:
+    - nicoddemus
+    - williamjamir
```
Comment (williamjamir) [inline recipes/python-hookman/meta.yaml]: 👍

**Comment by nicoddemus** [comment]
@igortg @conda-forge/staged-recipes I believe this is ready for review/merge

**Comment by scopatz** [comment]
LGTM. just waiting on Circle

**Comment by nicoddemus** [comment]
Cicle just passed. 👍

**Pull request merged**